views/docs/console.md

```diff
@@ -52,6 +52,22 @@ Note that if you are using Asterisk, the format is slightly different. The examp
 AHN> Adhearsion::OutboundCall.originate 'SIP/user1'
 </pre>
 
+As well, if you need to place calls out into the outside world, you may get an error message 
+such as "I don't know how to authenticate..." if you try to call directly. 
```

Comment:
Would it be possible to include an example of the exact error message? I admit I'm not familiar with this one.